frontend/src/lib/lemon-ui/LemonButton/LemonButton.scss

```diff
@@ -115,6 +115,7 @@
             display: flex;
             flex-shrink: 0;
             place-items: center center;
+            height: 100%;
```

Comment:
I don't know what this will do. I'm fearing two-row buttons with stretched icons.